service/src/main/java/tds/config/repositories/impl/AssessmentWindowQueryRepositoryImpl.java

```diff
@@ -77,8 +76,8 @@ public AssessmentWindowQueryRepositoryImpl(final DataSource dataSource) {
                     .withWindowId(rs.getString("windowID"))
                     .withMode(rs.getString("mode"))
                     .withWindowMaxAttempts(rs.getInt("windowMax"))
-                    .withStartTime(mapTimeStampToInstant(rs, "startDate"))
-                    .withEndTime(mapTimeStampToInstant(rs, "endDate"))
+                    .withStartTime(mapTimeStampToJodaInstant(rs, "startDate"))
```

Comment:
I know this is in common and I think I mentioned it there as well.  I'd like to be consistent on the naming regarding "Timestamp" vs "TimeStamp" in the method name.  I personally vote for "Timestamp" since that is the name of the class itself.  Not a blocker but that capital S jumps out at me :)